src/test/java/za/ac/sun/cs/search/adversarialagent/TestNegaDeepAgent.java

```diff
@@ -0,0 +1,20 @@
+package za.ac.sun.cs.search.adversarialagent;
+
+import org.junit.Test;
+import za.ac.sun.cs.adversarial.agent.Agent;
+import za.ac.sun.cs.adversarial.agent.NegaDeepAgent;
+import za.ac.sun.cs.adversarial.agent.RandomAgent;
+import za.ac.sun.cs.adversarial.referee.Referee;
+
+public class TestNegaDeepAgent {
+
+    @Test
+    public void smokeTest() {
```

Comment:
`smokeTest` kills the TT. Looks like the way we are trying to bound the Zobrist hashes is going negative on us. @Zhunaidm Give it a look when you get a chance! :+1: